packages/homepage/content/docs/8-ci.md

```diff
@@ -0,0 +1,73 @@
+---
+title: CI
+authors: ['Ives van Hoorne']
+description:
+  Here's what CodeSandbox CI is, how it will benefit you as a library maintainer, and how to set it up for your library.
+---
+
+## What is CodeSandbox CI?
+
+CodeSandbox CI is a [GitHub app](https://github.com/apps/codesandbox) that you can install in your repository. The app is responsible for building your library. Whenever a Pull Request is opened it builds a version of the library from that PR. This is published to our registry, so you can immediately test the library in CodeSandbox or locally without publishing to nom or elsewhere.
+
+While Client Sandboxes have faster previews and offline availability, they lack
```

Comment:
This one is still from the old config methinks.

Reply:
I only know of https://u2edh.csb.app/ - if you've updated somewhere else, then I'll need a link to it.